sdk/core/azure-core-amqp/test/ut_uamqp/uamqp_value_tests.cpp

```diff
@@ -222,18 +221,19 @@ TEST_F(TestValue, SimpleCreate)
   }
 
   {
-    AmqpValue descriptor(235ll);
+    AmqpValue descriptor(static_cast<uint64_t>(237ll));
```

Comment:
You changed 235 to 237 - it does not matter which one it is, correct?

Reply:
I wanted two different values for the descriptor number.

A described type (or a composite type), the descriptor is a 64bit number which uniquely describes the value. FOr this test, the number is irrelevant. For real code, there is a list of defined descriptor values in one of the headers.